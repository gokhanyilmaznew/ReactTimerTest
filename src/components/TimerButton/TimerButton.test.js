import React from 'react';
import TimerButton from './TimerButton';
import renderer from 'react-test-renderer';

describe("TimerButton component snapshot", () => {
    test("should create snapshot", () => {
    
        expect().toMatchSnapshot();
    });
});