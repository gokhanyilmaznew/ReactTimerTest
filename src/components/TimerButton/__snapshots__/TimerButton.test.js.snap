// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TimerButton component snapshot should create snapshot 1`] = `undefined`;

exports[`Todo component snapshot should create snapshot 1`] = `undefined`;
